cli/main.py

```diff
@@ -1,22 +1,22 @@
 #!/usr/bin/env python3
```

Comment:
Wait... I guess, cli should be a module (so it should have `__init__.py`). Also, it should probably live in nucypher namespace.

Currently, I am installing in virtualenv (so no Pipenv to help us, like in real world), and:
```
  File "/home/michwill/Projects/nucypher-kms/_test/bin/nucypher", line 7, in <module>
    from cli.main import cli
ModuleNotFoundError: No module named 'cli'
```

Reply:
Yeah, I think so too.  But not today.